src/styles/mixins/effects.scss

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
`effects` is perhaps ambiguous. Perhaps transitions.scss or animations.scss might be more obvious?

Reply:
Sure, I like animations. We may end up with some keyframe stuff eventually too.